narwhal/consensus/src/bullshark.rs

```diff
@@ -87,7 +88,8 @@ impl ConsensusProtocol for Bullshark {
         self.max_inserted_certificate_round = self.max_inserted_certificate_round.max(round);
 
         // Try to order the dag to commit. Start from the highest round for which we have at least
-        // f+1 certificates. This is because we need them to reveal the common coin.
+        // f+1 certificates. This is because we need them to reveal the common coin, or provide
```

Comment:
```suggestion
        // f+1 certificates. This is because we need them to provide
```

The mention of common coin comes from Tusk (there is no need for common coin in Bullshark)